tools/configen/configen/templates/sample_config.yaml

```
@@ -1,5 +1,6 @@
 defaults:
   - configen_schema
+  - _self_  
```

```suggestion
  - _self_
```
I think this is causing a linter failure :)